app/javascript/packs/ping_thing/index_template.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div>
```

Comment:
I would split this view into few smaller ones (eg. header section, chart section and table section). It would also eliminate the need to have this kind of empty `<div>` which makes no sense from html structure perspective (eg. the first section already has its own wrapper called `<div class="page-header">`).